starknet_py/net/account/account.py

```diff
@@ -53,8 +53,12 @@
 class Account(BaseAccount):
     """
     Default Account implementation.
+
+    :ivar ESTIMATED_FEE_MULTIPLIER: Amount by which each estimated fee is multiplied when using `auto_estimate`.
     """
 
+    ESTIMATED_FEE_MULTIPLIER: float = 1.5
```

Comment:
Wouldn't the docs for this generate correctly if you just added a docstring directly below `ESTIMATED_FEE_MULTIPLIER`, without the `:ivar ESTIMATED_FEE_MULTIPLIER:`?

That's how it worked with dataclasses.

Reply:
I think it wouldn't show at all in the class

Reply:
```py
ESTIMATED_FEE_MULTIPLIER: float = 1.5
"""Amount by which each estimated fee is multiplied when using `auto_estimate`."""
```
should work and display as 
![image](https://user-images.githubusercontent.com/52135326/215741080-02f5ee16-1a66-4b0d-97a7-903dc41b0c25.png)